.github/workflows/nightly-build.yml

```diff
@@ -88,3 +89,32 @@ jobs:
       - uses: docker/setup-buildx-action@v1
       - name: Push docker image
         run: ./mvnw -am -pl distribution/proxy -B -Prelease,docker.buildx.push -DskipTests -Dproxy.image.repository=${{ env.PROXY }} -Dproxy.image.tag=${{ github.sha }} clean install
+
+  build-proxy-native-image:
+    if: github.repository == 'apache/shardingsphere'
+    timeout-minutes: 90
+    permissions:
+      contents: read
+      packages: write
+    name: GraalVM Native Image on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: 'releases'
```

Comment:
- I just wanted to ask, `graalvm/setup-graalvm@v1` doesn't seem to have this keyword?  `version` only has `latest`, `dev`, or specifies a version like `22.2.0`.  Reference https://github.com/graalvm/setup-graalvm . 👀